client/client.go

```diff
@@ -371,11 +377,26 @@ type (
 		//  - serviceerror.NotFound
 		DescribeTaskQueue(ctx context.Context, taskqueue string, taskqueueType enumspb.TaskQueueType) (*workflowservice.DescribeTaskQueueResponse, error)
 
-		// ResetWorkflowExecution reset an existing workflow execution to WorkflowTaskFinishEventId(exclusive).
+		// ResetWorkflowExecution resets an existing workflow execution to WorkflowTaskFinishEventId(exclusive).
 		// And it will immediately terminating the current execution instance.
 		// RequestId is used to deduplicate requests. It will be autogenerated if not set.
 		ResetWorkflowExecution(ctx context.Context, request *workflowservice.ResetWorkflowExecutionRequest) (*workflowservice.ResetWorkflowExecutionResponse, error)
 
+		// UpdateWorkerBuildIDOrdering **IN DEVELOPMENT**
+		// Allows you to update the worker-build-id based version graph for a particular task queue. This is used in
+		// conjunction with workers who specify their build id and thus opt into the feature. For more, see: <doc link>
+		//  - workerBuildId is required and indicates the build id being added to the version graph.
+		//  - previousCompatible may be empty, and if set, indicates an existing version the new id should be considered
+		//   compatible with.
+		//  - If becomeDefault is true, this new id will become the default version for new workflow executions.
+		UpdateWorkerBuildIDOrdering(ctx context.Context, taskQueue string, workerBuildId string, previousCompatible string, becomeDefault bool) error
```

Comment:
It's not necessarily just for adding, as it can rewire existing nodes. I want to keep the names close to the actual API names as well, since we do that everywhere else